src/Compilers/Core/CodeAnalysisTest/Emit/CustomDebugInfoTests.cs

```diff
@@ -369,6 +371,222 @@ public void EncCdiAlignment()
         }
 
         [Fact]
+        public void UsingInfo1()
+        {
+            var builder = new BlobBuilder();
+            var cdiEncoder = new CustomDebugInfoEncoder(builder);
+            cdiEncoder.AddUsingGroups(new int[0]);
+            var cdi = cdiEncoder.ToArray();
+
+            Assert.Equal(0, cdiEncoder.RecordCount);
+            Assert.Null(cdi);
+        }
+
+        [Fact]
+        public void UsingInfo2()
+        {
+            var builder = new BlobBuilder();
+            var cdiEncoder = new CustomDebugInfoEncoder(builder);
+            cdiEncoder.AddUsingGroups(new[] { 1, 2, 3, 4 });
+            var cdi = cdiEncoder.ToArray();
+
+            Assert.Equal(1, cdiEncoder.RecordCount);
+
+            AssertEx.Equal(new byte[]
+            {
+                0x04,       // version
+                0x01,       // record count
+                0x00, 0x00, // alignment
+
+                0x04, // version
+                0x00, // record kind
+                0x00,
+                0x00,
+
+                // aligned record size
+                0x14, 0x00, 0x00, 0x00,
+                
+                // payload (4B aligned)
+                0x04, 0x00, // bucket count
+                0x01, 0x00, // using count #1
+                0x02, 0x00, // using count #2
+                0x03, 0x00, // using count #3
+                0x04, 0x00, // using count #3
+                0x00, 0x00  // alignment
+            }, cdi);
+        }
+
+        [Fact]
+        public void UsingInfo3()
+        {
+            var builder = new BlobBuilder();
+            var cdiEncoder = new CustomDebugInfoEncoder(builder);
+            cdiEncoder.AddUsingGroups(new[] { 1, 2, 3 });
+            var cdi = cdiEncoder.ToArray();
+
+            Assert.Equal(1, cdiEncoder.RecordCount);
+
+            AssertEx.Equal(new byte[]
+            {
+                0x04,       // version
+                0x01,       // record count
+                0x00, 0x00, // alignment
+
+                0x04, // version
+                0x00, // record kind
+                0x00,
+                0x00,
+
+                // aligned record size
+                0x10, 0x00, 0x00, 0x00,
+                
+                // payload (4B aligned)
+                0x03, 0x00, // bucket count
+                0x01, 0x00, // using count #1
+                0x02, 0x00, // using count #2
+                0x03, 0x00, // using count #3
+            }, cdi);
+        }
```

Comment:
Yes, this one has just 3 groups and no alignment. The previous has 4 groups and alignment.